src/os/freertos/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ find_package(FreeRTOS COMPONENTS
             NONE
             REQUIRED)
 
+get_target_property(PROPOLIS_OS_COMMON_INCLUDES SwarmUS::Propolis::OS::Common INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
Careful, you're overriding the PR fix we did before

Reply:
good catch, thanks for pointing it